vertica_python/vertica/cursor.py

```diff
@@ -377,6 +378,8 @@ def format_operation_with_parameters(self, operation, parameters, is_csv=False):
 
                 if isinstance(param, string_types):
                     param = self.format_quote(as_text(param), is_csv)
+                elif isinstance(param, datetime.datetime):
```

Comment:
Same logic should be added under line 398.

Reply:
Done

Reply:
Thanks. There are three datatypes that need to be quoted: DATE(datetime.date), DATETIME(datetime.datetime), and TIME(datetime.time).